locales/en.json

```diff
@@ -182,11 +190,6 @@
   "kebab.exportAsJson": "Export as JSON",
   "kebab.exportAsPDF": "Export as PDF",
   "knowledgebasearticle": "Knowledgebase article",
-  "knownExploit": "Has a known exploit",
```

Comment:
that's weird. Can you run once again `npm run translations` . known exploit should be included in en.json

Reply:
Ah, I see, I just ran `npm run translations:compile`